recipes/pymde/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
If you bump to `0.1.5`, you can remove the license file since it's now included in the PyPI tarball.